imports/ui/pages/professor/manage_course.jsx

```diff
@@ -137,8 +138,15 @@ class _ManageCourse extends Component {
 
   toggleAllowStudentQuestions () {
     Meteor.call('courses.toggleAllowStudentQuestions', this.props.course._id, (error) => {
-      if (error) return alertify.error('Error allowing/refusing student questions ' + error.error)
-      alertify.success('Students ' + (this.props.course.allowStudentQuestions ? 'can' : 'cannot') + ' submit questions')
+      if (error) alertify.error('Error allowing/refusing student questions ' + error.error)
+      else alertify.success('Students ' + (this.props.course.allowStudentQuestions ? 'can' : 'cannot') + ' submit questions')
+    })
+  }
+
+  toggleRequireApprovedQuestions () {
```

Comment:
toggleRequireApprovedPublicQuestions - in general everywhere, make sure that it's explicit that we are toggling whether public questions in a course require approval.